documentation/modules/cruise-control/proc-interacting-with-cruise-control-api.adoc

```diff
@@ -0,0 +1,9 @@
+// This module is included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-interacting-with-cruise-control-api-{context}']
+= Interacting with the Cruise Control API
+
+TBD
```

Comment:
Same as abive

Reply:
Commented this out in `assembly-cruise-control-concepts.adoc` so it doesn't show up in the docs